@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -306,4 +281,46 @@ describe("MapView", function() {
             }
         }
     });
+
+    it("updates background tiling scheme", async function() {
+        this.timeout(500);
+        let setupFrameCallback: FrameRequestCallback = () => {};
+
+        let customMapView: MapView;
+
+        if (inNodeContext) {
+            global.requestAnimationFrame = (callback: FrameRequestCallback) => {
+                const callbackWrapper = () => {
```

Comment:
I would wrap all of this in `setTimeout(..., 0)` or `process.nextTick` so it is really async as real `requestAnimationFrame. is.

Reply:
done.